reporter/build.gradle

```diff
@@ -22,4 +22,10 @@ dependencies {
 
     implementation 'org.apache.poi:ooxml-schemas:1.4'
     implementation 'org.apache.poi:poi-ooxml:3.17'
+
+    implementation "org.jetbrains.kotlinx:kotlinx-html-jvm:$kotlinxHtmlVersion"
+
+    // This is required to not depend on the version of Apache Xalan bundled with the JDK. Otherwise the formatting of
+    // the HTML generated in StaticHtmlReporter is slightly different with different Java versions.
+    implementation 'xalan:xalan:2.7.2'
```

Comment:
Should we move this version to a property? Even if I just realized that for some reason we didn't do this for the other Apache libraries above...

Reply:
Done.